packages/main/src/ComboBox.ts

```diff
@@ -402,7 +402,7 @@ class ComboBox extends UI5Element implements IFormInputElement {
 	 * **Note:** If not specified, a default text (in the respective language) will be displayed.
 	 *
 	 * **Note:** The `valueStateMessage` would be displayed,
-	 * when the `ui5-combobox` is in `Information`, `Warning` or `Error` value state.
+	 * when the `ui5-combobox` is in `Information`, `Critical` or `Negative` value state.
 	 * @since 1.0.0-rc.9
 	 * @public
 	 */
```

Comment:
Additionally, Also I think in the TextArea in some of the jsdoc, it's used Error instead of Negative:
`the component turns into "Error"`

Reply:
Done.